ball_pit/ball_pit.ino

```diff
@@ -295,7 +295,13 @@ void increment_num_balls() {
   } else if (num_balls % 100 == 0) {
     play_short_tune();
   } else {
-    beep();
+    if (is_power_of_two(num_balls)) {
```

Comment:
Why are these in the `else` block? Just chain them as `else if`s; it's functionally the same and a bit nicer to read.